2016/05/09 08:30:57 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(UserWeb), 'You are not aut...')
#1 C:\xampp\htdocs\yii\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('User')
#9 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#11 {main}
REQUEST_URI=/siadm/index.php/User
---
